frontend/utilities/constants.ts

```diff
@@ -159,6 +159,16 @@ export const DEFAULT_POLICIES = [
       "Contact your IT administrator to help you enroll your computer in your organization's MDM. If already enrolled, ask your IT administrator to enable the screen lock feature in the profile configuration.",
     platform: "darwin",
   },
+  {
+    key: 15,
+    query:
+      "SELECT 1 FROM registry WHERE path = 'HKEY_LOCAL_MACHINE\Software\Microsoft\Windows\CurrentVersion\Policies\System\InactivityTimeoutSecs' AND CAST(data as INTEGER) <= 1800;",
+    name: "Screen lock enabled (Windows)",
+    description: "Checks if the screen lock is enabled and configured to lock the system within 30 minutes or less.",
+    resolution:
+      "Ask your administrator to enable the Interactive Logon: Machine inactivity limit setting with a value of 1800 seconds or lower.",
```

Comment:
"IT administrator" would be clearer to an end user. They probably don't know where to find an "administrator." Might end up going to the reception person haha. "IT" would tell them to go to the IT dept at least. 

Is this the same concept where we are checking the MDM setting, not necessarily checking the setting itself? I'm wondering because the resolution step here seems focused on getting your admin to do the MDM setting. You can't do it yourself?

Reply:
I'll update the text. In this case we are checking the actual setting, but there's many ways of configuring a screen saver on Windows. It can be per device (which is what we are checking) - which is usually done either by a GPO or MDM. If we told a user to configure it manually, that would be a per-user setting that only applies to that one person on that one machine.